dim/src/errors.rs

```diff
@@ -47,6 +47,8 @@ pub enum DimError {
     InvalidCredentials,
     #[error(display = "Requested username is not available.")]
     UsernameNotAvailable,
+    #[error(display = "Paths are already added to a library")]
+    AlreadyAddedToLibrary { duplicates: Vec<String> },
```

Comment:
This should be renamed to something else. ATM this implies that the Paths have already been added to a library rather than the database.

Reply:
I think, this is what is happening.